services/registry.dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:12
+RUN addgroup -g 1000 -S node && \
```

Comment:
If the process needs to write to the disk, using a limited user like this might not work.